check.sh

```diff
@@ -0,0 +1,13 @@
+echo "Cloning opentelemetry-proto folder"
+svn checkout https://github.com/open-telemetry/opentelemetry-proto/trunk/opentelemetry
+mkdir proto
+protoc --proto_path=./ --php_out=proto opentelemetry/proto/collector/logs/v1/logs_service.proto
```

Comment:
We should also verify that the end user has protoc available in the same vein as my first comment.